src/XrdAccSciTokens.cc

```diff
@@ -286,6 +290,52 @@ class XrdAccSciTokens : public XrdAccAuthorize
         return result ? AddPriv(oper, XrdAccPriv_None) : OnMissing(Entity, path, oper, env);
     }
 
+    virtual  Issuers IssuerList() override
+    {
+        /*
+        Convert the m_issuers into the data structure:
+        struct   ValidIssuer
+        {std::string issuer_name;
+         std::string issuer_url;
+        };
+        typedef std::vector<ValidIssuer> Issuers;
+        */
+        Issuers issuers;
+        for (auto it: m_issuers) {
```

Comment:
I presume the issuer list never changes and the caller may hold on to the list for future use even different clients, yes? If so, that should be noted in the header comments.